File: questao10/python/calculadora.py
class Calculadora:
    # Método para somar dois números
    def somar_dois(self, a, b):
        return a + b

    # Método para somar três números
    def somar_tres(self, a, b, c):
        return a + b + c


# Testando a Calculadora
calc = Calculadora()

# Somando dois números
print("Soma de 2 números:", calc.somar_dois(5, 10))

# Somando três números
print("Soma de 3 números:", calc.somar_tres(5, 10, 15))

File: questao10/go/calculadora.go
package main

import "fmt"

// Função para somar dois números
func somarDois(a int, b int) int {
    return a + b
}

// Função para somar três números
func somarTres(a int, b int, c int) int {
    return a + b + c
}

func main() {
    // Somando dois números
    fmt.Println("Soma de 2 números:", somarDois(5, 10))

    // Somando três números
    fmt.Println("Soma de 3 números:", somarTres(5, 10, 15))
}
